homeassistant/components/switch/rainmachine.py

```diff
@@ -176,16 +246,16 @@ def _update(self) -> None:
         except exceptions.HTTPError as exc_info:
             _LOGGER.error('Unable to update info for program "%s"',
                           self.rainmachine_id)
-            _LOGGER.debug(str(exc_info))
+            _LOGGER.debug(exc_info)
 
 
 class RainMachineZone(RainMachineEntity):
     """A RainMachine zone."""
 
-    def __init__(self, client, zone_json, **kwargs):
+    def __init__(self, client, zone_json, zone_run_time, **kwargs):
         """Initialize a RainMachine zone."""
-        super(RainMachineZone, self).__init__(client, zone_json, **kwargs)
-        self._run_time = kwargs.get(CONF_ZONE_RUN_TIME)
+        super().__init__(client, zone_json, **kwargs)
+        self._run_time = kwargs.get(zone_run_time)
```

Comment:
This change is already pushed:

<img width="622" alt="screen shot 2017-08-07 at 1 26 53 pm" src="https://user-images.githubusercontent.com/47216/29042447-23ab2d3a-7b74-11e7-904e-5455d7805f03.png">

Are you not seeing that? Wondering if I messed up the branch further...